### ISSUE TITLE
Using openstreetmap instead of google maps

### ISSUE THREAD

**Issue by Uatschitchun** [studio-arrenberg/quartiersplattform#154]
There's https://github.com/mcguffin/acf-openstreetmap-field

As if using google maps there's a fee to pay, and google is a "Datenkrake", for locations and maps one could switch to using OSM and leaflet.js?

**Comment by j0hannr** [comment]
That would be really nice! 

The Repository is pretty old did you already try it with acf?

**Comment by Uatschitchun** [comment]
No, just stumbled upon and didn't want to forget about it. But leaflet is something I've used already here:
https://github.com/Uatschitchun/Latrine-live
